modules/apps/item-selector/item-selector-taglib/src/main/java/com/liferay/item/selector/taglib/servlet/taglib/GroupSelectorTag.java

```diff
@@ -114,13 +115,18 @@ private Group _getGroup(ThemeDisplay themeDisplay) {
 		int[] startAndEnd = SearchPaginationUtil.calculateStartAndEnd(
 			cur, delta);
 
+		_groups = new ArrayList<Group>();
+		_groups.add(group);
+
+		/*
 		_groups = groupItemSelectorProviderOptional.map(
 			groupItemSelectorProvider -> groupItemSelectorProvider.getGroups(
 				group.getCompanyId(), group.getGroupId(), keywords,
 				startAndEnd[0], startAndEnd[1])
 		).orElse(
 			Collections.emptyList()
 		);
+		*/
 
```

Comment:
I am able to limit the list of selectable sites in the breadcrumbs to be that of the portlet-configured scope. However, I am unsure if this is the correct approach, as this does not require the use of the scope's type and classPK.

Reply:
Looks like we need to use the scope and type in case we are scoped to a layout, not just a site.  From the PTR ticket:

> We need this because we could also select the current layout (not only the current site)